config/software/td-agent-files.rb

```diff
@@ -9,7 +9,7 @@
 build do
   block do
     # setup related files
-    pkg_type = project.packagers_for_system.first.id.to_s
+    pkg_type = project.packager.id.to_s
```

Comment:
The "packagers_for_system" functions only exist in Omnibus 5.5.0 as a replacement of packager.